src/components/Details.js

```diff
@@ -1,8 +1,30 @@
-export default function Detail({ movie, onReturnHome }) {
-    return (
-      <div className="movie-detail">
-        <button onClick={onReturnHome}>Return Home</button>
+import React from 'react';
+import Circle from './Circle';
+import './Details.css';
+
+export default function Details({ movie, onReturnHome }) {
+  return (
+    <div className="movie-details">
```

Comment:
I suggest rethinking the HTML structure of this component to ensure it's semantically correct. Using <main> and <section> tags could help make this more semantic.